resources/virt_test.go

```diff
@@ -46,3 +46,22 @@ func TestExpandHome(t *testing.T) {
 		}
 	}
 }
+
+func TestNumToAlpha(t *testing.T) {
+	var NumToAlphaTests = []struct {
+		number int
+		result string
+	}{
+	{0, "a"},
+	{25, "z"},
+	{26, "aa"},
+	{27, "ab"},
```

Comment:
please add a test for "aaa" and "aab", and maybe something longer like "cool". Might as well show off now that you wrote this lovely function :)